packages/utils/src/getLocaleSeparators.native.ts

```diff
@@ -0,0 +1,27 @@
+// on iOS we can't use formatToParts, so we need to find the separators differently
+// maybe we can use for this for desktop too and don't have two implementations?
+export function getLocaleSeparators(locale: string): {
+    decimalSeparator: string;
+    thousandsSeparator: string;
+} {
+    // Format a number with both thousands and decimal parts
+    const formattedNumber = new Intl.NumberFormat(locale).format(1234567.89);
```

Comment:
Wouldn't be nice to be a bit more defensive in case there is a wrong locale it will not throw error or do we prefer it to throw error when it fails. Like the code below will throw an error.

```
new Intl.NumberFormat('de-DEW').format(1234567.89);
```